prow/plugins/verify-owners/verify-owners.go

```diff
@@ -138,50 +137,51 @@ func handle(ghc githubClient, gc *git.Client, log *logrus.Entry, pre *github.Pul
 		}
 		// Check labels against blacklist
 		if sets.NewString(labels...).HasAny(labelsBlackList...) {
-			wrongOwnersFiles[f] = fmt.Errorf("OWNERS file contains blacklisted labels: %s", sets.NewString(simple.Config.Labels...).Intersection(sets.NewString(labelsBlackList...)).List())
+			wrongOwnersFiles[f] = fmt.Sprintf("File contains blacklisted labels: %s.", sets.NewString(labels...).Intersection(sets.NewString(labelsBlackList...)).List())
 			continue
 		}
 		// Check approvers isn't empty
 		if filepath.Dir(f) == "." && len(approvers) == 0 {
-			wrongOwnersFiles[f] = errors.New("no approvers in the root directory OWNERS file")
+			wrongOwnersFiles[f] = "No approvers defined in file."
```

Comment:
I think it is important to specify that the problem is that the **root** OWNERS file does not have approvers. The new message suggests that all OWNERS files must have approvers defined which is not the case.

Reply:
Indeed, but the rationale was that the comment was to be attached to the proper file... Maybe I will specify that root OWNERS files must have approvers.

Reply:
I think we should still specify that it is the root file to make it clear that the requirement only applies to that file.